src/main/java/org/broadinstitute/hellbender/tools/spark/sv/utils/GATKSVVCFHeaderLines.java

```diff
@@ -90,6 +93,11 @@ private static void addSymbAltAlleleLine(final VCFSimpleHeaderLine line) {
         addInfoLine(new VCFInfoHeaderLine(GATKSVVCFConstants.DUPLICATION_NUMBERS, VCFHeaderLineCount.R, VCFHeaderLineType.Integer, "Number of times the sequence is duplicated on reference and on the alternate alleles"));
         addInfoLine(new VCFInfoHeaderLine(GATKSVVCFConstants.DUP_ANNOTATIONS_IMPRECISE, 0, VCFHeaderLineType.Flag, "Whether the duplication annotations are from an experimental optimization procedure"));
 
+        addInfoLine(new VCFInfoHeaderLine(GATKSVVCFConstants.INVDUP_STRANDS, VCFHeaderLineCount.A, VCFHeaderLineType.String,
+                "Strands of the duplicated sequence on alt allele, one group for each alt allele (currently only available for inverted duplication variants)"));
+        addInfoLine(new VCFInfoHeaderLine(GATKSVVCFConstants.INV_TRANS_INS_REF_SPAN, VCFHeaderLineCount.UNBOUNDED, VCFHeaderLineType.String,
```

Comment:
While changing the code for re-using this annotation, I realized that the case is a bit more complicated than what we thought of:
This annotation is for the area that is NOT covered by the alignment ref span overlap
```
two              <---------------------
one ------------------------>
                             |||||||||||
                             these bar bases are the inv ins annotated part
```
But there could be a part of the contig that is not at all covered by these two alignments, i.e. that part of the contig is unmapped. Such sequence would go into the annotation `INSERTED_SEQUENCE`, and without any `INSERTED_SEQUENCE_MAPPINGS`.
And for the case when there's 3 alignments (not the case being covered here but will be in the graph-based cpx case), where the 1st and last are like the case here, with the middle one mapped somewhere else on the reference, we need these two different annotations.

Do you agree?

Reply:
Do we allow multiple "INSERTED_SEQUENCE" values? If so I'd think this would still be fine, right? What if we made a 1-to-1 mapping between INSERTED_SEQUENCE values and INSERTED_SEQUENCE_MAPPINGS values, and gave the ones without a mapping a value of "." or something to indicate that it doesn't have a mapping? Would that solve the issue?

Reply:
I like this proposal better, so I am going to implement it in this PR in the next commit.

To make it more precise, based on our discussions this morning, and the fact that some of the insertion mappings are "extracted" from a larger alignment block, I'm going to strip down the mapping information stored in the INSERTED_SEQUENCE_MAPPINGS, because 
1) the extracted info could not make guarantee that MQ, NM is correct; and 
2) if the analyst is keen on the exact mapping and alignment, he/she can always take the provided inserted sequence and the given mapping location and run alignment with his/her choice of alignment parameters.

So the information stored in INSERTED_SEQUENCE_MAPPINGS will be formatted as

`refChr_startInclusive_endInclusive_ORIENTATION_O/E`

where the `O/E` is to signal if the mapping location is extracted from a bigger alignment block or not.

What do you think?